src/main/java/Duke.java

```diff
@@ -6,5 +7,42 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        String line;
+        Scanner in = new Scanner(System.in);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can i do for you?");
+        String[] List = new String[100];
+        int ListNo = 0;
+        do{
+            line = in.nextLine();
+            if (line.equals("bye")){
+                break;
+            }else if(line.compareTo("list") != 0) {
+                if(line.contains("mark")){
+                    int i = line.indexOf(" ") + 1;
+                    Task t = new Task(List[Integer.parseInt(line.substring(i))-1]);
+                    t.markAsDone();
+                    System.out.println("Nice! I've marked this task as done:");
+                    System.out.println("[" + t.getStatusIcon() + "] " + List[Integer.parseInt(line.substring(i))-1]);
+                } else if(line.contains("unmark")) {
+                    int i = line.indexOf(" ") + 1;
```

Comment:
I'm not quite able to decipher why we are counting the index of the space, maybe might want to add a comment to describe this part.